.github/workflows/pr-workflow.yml

```
@@ -8,7 +8,28 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build-deployment:
+
+  non-server-feature:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)